source/gameengine/Ketsji/KX_FontObject.cpp

```diff
@@ -105,6 +106,32 @@ KX_FontObject::~KX_FontObject()
 	//it's handled in KX_Scene::NewRemoveObject
 }
 
+const MT_Vector2 KX_FontObject::GetTextDimensions()
+{
+	float width = 0, height = 0;
```

Comment:
Follow the blender code style about float write; use 0.0f same think for  / 10 --> / 10.0f